packages/core/src/data-access/commits.ts

```diff
@@ -26,13 +26,14 @@ export async function findHeadCommit(
 
 export type FindCommitProps = {
   uuid: string
-  projectId: number
+  projectId?: number
 }
 export async function findCommit(
   { projectId, uuid }: FindCommitProps,
   tx = database,
 ): Promise<TypedResult<Commit, LatitudeError>> {
-  if (uuid === HEAD_COMMIT) return findHeadCommit({ projectId }, tx)
+  if (uuid === HEAD_COMMIT && projectId)
+    return findHeadCommit({ projectId }, tx)
```

Comment:
Brackets

Reply:
that's prettier better not to touch it

Reply:
Prettier doesn't complain if you put the brackets

Reply:
Sorry what I mean is that we cannot rely on andres linter to catch all prettier formattings on PRs that we don't like we should change the prettier config if we don't like it.